at_client/src/main/java/org/atsign/client/util/RegisterUtil.java

```diff
@@ -183,10 +243,41 @@ public String validateOtp(String email, AtSign atsign, String otp, String regist
         }
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Please move this to the line-of-code level rather than method-level

Reply:
I have made this change as part of the latest commit